ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts/wan.py

```diff
@@ -30,3 +30,19 @@ def wan_path_groups(self: EosDesignsFacts) -> list | None:
             return None
 
         return self.shared_utils.wan_local_path_groups
+
+    @cached_property
+    def wan_ha_interfaces(self: EosDesignsFacts) -> str | None:
+        return self.shared_utils.wan_ha_interfaces if self.shared_utils.wan_ha else None
+
+    @cached_property
+    def wan_ha_peer(self: EosDesignsFacts) -> str | None:
+        return self.shared_utils.wan_ha_peer if self.shared_utils.wan_ha else None
```

Comment:
The check should be in the shared_utils instead of this.

Reply:
facts gone !